Intersect.Client/Interface/Game/DescriptionWindows/ItemDescriptionWindow.cs

```diff
@@ -354,19 +354,21 @@ protected void SetupEquipmentInfo()
                 else
                 {
                     _ = mItem.TryGetRangeFor((Stat)i, out var range);
```

Comment:
You don't need a separate null check, the boolean being discarded here will already tell you that